### PR TITLE
12.0 payments

### PR DESCRIPTION
**Pull request by mclaeysb**

### CODE REVIEW

File: logeco_custom/__manifest__.py [hunk 1]
```diff
@@ -10,8 +10,20 @@
...
+        "housing_cooperative_base",
+        "l10n_ch_states",
+        "l10n_ch_payment_slip",
+        "l10n_ch_pain_credit_transfer",
+    ],
+    "data": [
+        "data/data.xml",
+        "views/building.xml",
+        "views/housing.xml",
+        "demo/demo.xml",  # fixme
```
Comment (robinkeunen) [inline logeco_custom/__manifest__.py:23]: don't forget this

File: logeco_custom/hooks.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Inspiration from OCA/project/project_task_code
+from odoo import api, SUPERUSER_ID
+import logging
+
+
+_logger = logging.getLogger(__name__)
+
+
+def pre_init_hook(cr):
```
Comment (robinkeunen) [inline logeco_custom/hooks.py:9]: What is this _sorcery_? Can you explain on monday? :-)

**Review by robinkeunen — COMMENTED** [review]
Again, the description is a bit terse so I'm not sure what I'm reviewing :-)

**Comment by mclaeysb** [comment]
This code implements all tasks related to payments. 
(I'll try to comment PR's a bit more)

File: logeco_custom/__manifest__.py (hunk 1 above)
Comment (mclaeysb) [reply logeco_custom/__manifest__.py:23]: It's on my wrapping up todo list, but tbh I don't know what to fix here. Does this refer to the 'noupdate'? I thought I should move this to the `demo/` directory, but that makes it crash?

File: logeco_custom/hooks.py (hunk 2 above)
Comment (mclaeysb) [reply logeco_custom/hooks.py:9]: Pre init hooks, see comments for explanation. Live explanation available upon request. This is currently sub-optimal and require a manual db intervention, so I'd like to hear your thoughts!

File: logeco_custom/__manifest__.py (hunk 1 above)
Comment (robinkeunen) [reply logeco_custom/__manifest__.py:23]: It should me moved in the __manifest__ from the data to the  demo section:

```python
{
    "name": "Housing Cooperative Custom",
...
    ],
    "data": [
        "data/data.xml",
        "views/building.xml",
        "views/housing.xml",
   
    ],
    "demo": [
        "demo/demo.xml",
    ],
...
}
```

Comment (mclaeysb) [reply logeco_custom/__manifest__.py:23]: This gives an error for me. I'll look at it tomorrow..

**Pull request merged**